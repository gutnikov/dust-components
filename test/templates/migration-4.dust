<div>
    {@Container className="{containerClass}" headerText="{containerHeaderText}"}
        {@AuthorsList itemClass="{itemClass}" authors=authors/}
    {/Container}
</div>
