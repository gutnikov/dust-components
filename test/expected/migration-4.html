<div>
    <div class="container authors">
        <h1>Authors list:</h1>
        <div class="container-body">
            <div class="authors-list">
                <div class="author authors-item">
                    <div class="author-name">Dan</div>
                    <div class="author-surname">Brown</div>
                    <button class="button button-vote">Like!</button>
                </div>
                <div class="author authors-item">
                    <div class="author-name">Steven</div>
                    <div class="author-surname">King</div>
                    <button class="button button-vote">Like!</button>
                </div>
            </div>
        </div>
    </div>
</div>
